kart/raster/v1.py

```diff
@@ -276,6 +298,35 @@ def get_tile_summary_from_pointer_blob(
             result["pamSize"] = pam_summary["size"]
         return result
 
+    def is_tile_compatible(self, ds_format, tile_summary):
+        tile_format = tile_summary["format"]
+        if isinstance(ds_format, dict):
```

Comment:
when might this not be a dict?

Reply:
ah... maybe never, but there's a few methods like this that need to be given something like "target format". Depending on where they were used, it was sometimes more useful to have them accept the "format.json" meta item, or sometimes the little format-summary that we store per tile eg "geotiff/cog" or "laz-1.4/copc-1.0" but after a while I tried to make them accepting of either one, where possible.